iothub/device/src/Transport/AmqpIot/AmqpIotTransportLog.cs

```diff
@@ -23,7 +23,7 @@ public override void AmqpAddSession(object source, object session, ushort localC
         {
             if (Logging.IsEnabled)
             {
-                Logging.Associate(source, session, "AmqpAddSession:", $"{" source:"}{source}.{" session:"}{session}.{" localChannel:"}.{localChannel}.{" remoteChannel:"}.{remoteChannel}");
```

Comment:
I just... can't...

We string interpolate and include some string literals, but wrap them in curly braces to format in?